copy.lua

```diff
@@ -0,0 +1,417 @@
+--designating tool
+
+--[====[
+
+copy
+====
+copy is a gui program designed to make designating easier and more efficient.
+
+]====]
+
+--[[
+ Author's Note:
+ 
+ shoutout to twitch.tv/keupo
+ created by flavorstreet
+ 
+ pipicaca.
```

Comment:
This is fine - I'm less enthusiastic about people adding their names to every script they touch, since that's what Git history is for, but background information like this can be useful to have. (Not sure what significance `pipicaca` has here, but not an issue.)